core/src/scss/core/fonts/_fontawesome-fonts.scss

```diff
@@ -1,7 +1,7 @@
 @charset "UTF-8";
 
 @font-face {
-  font-family: 'Font Awesome 5 Free';
+  font-family: "Font Awesome 5 Free";
```

Comment:
FYI the way we include FontAwesome here, we don't have the "solid" set available - the "Free" set includes the solids and some of the "regulars", and the solids are in a separate file. For example, if you'd like to include the filled in version of the dismiss icon, we'll need to include the solid set in FontAwesome as well.

Reply:
![Screen Shot 2020-09-01 at 10 57 15 AM](https://user-images.githubusercontent.com/42749717/91889124-2c182d00-ec42-11ea-945a-a33438fb4d76.png)

This screenshot is from another project - we might need to add the solid ones to the asset folder for Decanter too if we use them.

Reply:
Thanks, I noticed the missing asset on our builds. Should be fixed now.